Configuration/StandardSequences/python/L1TrackTrigger_cff.py

```diff
@@ -5,13 +5,16 @@
 from L1Trigger.TrackerDTC.ProducerED_cff import *
 from L1Trigger.TrackFindingTracklet.L1HybridEmulationTracks_cff import * 
 
-L1TrackTrigger=cms.Sequence(TrackTriggerClustersStubs*TrackTriggerAssociatorClustersStubs*TrackerDTCProducer)
```

Comment:
@rekovic please have a look to the changes of this file. I'm removing two modules that were using the MCtruth in order to avoid errors such as 
```
----- Begin Fatal Exception 01-Jul-2020 12:13:37 CEST-----------------------
An exception of category 'ProductNotFound' occurred while
   [0] Processing  Event run: 1 lumi: 4 event: 303 stream: 0
   [1] Running path 'L1TrackTrigger_step'
   [2] Calling method for module TTClusterAssociator_Phase2TrackerDigi_/'TTClusterAssociatorFromPixelDigis'
Exception Message:
Principal::getByToken: Found zero products matching all criteria
Looking for type: edm::DetSetVector<PixelDigiSimLink>
Looking for module label: mix
Looking for productInstanceName: Tracker

   Additional Info:
      [a] If you wish to continue processing events after a ProductNotFound exception,
add "SkipEvent = cms.untracked.vstring('ProductNotFound')" to the "options" PSet in the configuration.

----- End Fatal Exception -------------------------------------------------
```